web/test/integration/project.feature

```diff
@@ -0,0 +1,11 @@
+Feature:  Save the project json in db 
+
+Scenario: Ensure the create public project
```

Comment:
The test here is not the create, that is the step def, the test is the reading. I would suggest something like `List public project on the homepage`

Reply:
okay will update